### PR TITLE
StackOverFlow search

### PR DESCRIPTION
**Pull request by judemont**
Hello,
I have added an option to search for [stackoverflow](https://stackoverflow.com) questions and display answers, using the [stackexchange api](api.stackexchange.com)

Screenshot from 2023-09-16 14-46-16
image

### CODE REVIEW

File: static/css/style.css [hunk 1]
```diff
@@ -1051,6 +1051,27 @@ p {
     font-size: 16px;
 }
 
+.answer_preview{
+    max-height: 500px;
+    overflow-y: scroll;
+    white-space: pre-wrap;
```
Comment (amogusussy) [inline static/css/style.css:1057]: This makes each paragraph have a large amount of spacing around it, which doesn't look good IMO.

Comment (amogusussy) [reply static/css/style.css:1057]: I'm talking about line 1057 in particular. I don't know why it highlighted all those other lines.

File: static/css/style.css [hunk 2]
```diff
@@ -1051,6 +1051,27 @@ p {
...
+.answer_preview pre {
+    white-space: pre-wrap;
+    padding: 10px;
+    padding-left: 40px;
+    border-radius: 10px;
+    background-color: var(--code-preview-background);
+    background-clip: padding-box;
+    width: 80%;
+}
+
```
Comment (amogusussy) [inline static/css/style.css:1074]: Add something like this to make the inline code snippets stand out more:
```
code:not(pre code) {
  background: var(--code-preview-background);
  padding: 1px 4px;
  border-radius: 3px;
}
```

Comment (Extravi) [reply static/css/style.css:1074]: i will be redoing this pr for speed etc i will also be adding something like this https://extravi.dev/tailsx/

Comment (Extravi) [reply static/css/style.css:1074]: image

File: static/css/style.css (hunk 1 above)
Comment (judemont) [reply static/css/style.css]: You're right, I removed the line.

File: static/css/style.css (hunk 2 above)
Comment (judemont) [reply static/css/style.css:1073]: > Add something like this to make the inline code snippets stand out more:
> 
> ```
> code:not(pre code) {
>   background: var(--code-preview-background);
>   padding: 1px 4px;
>   border-radius: 3px;
> }
> ```

It's done thanks

**Comment by MixoMax** [comment]
This looks great and would be very helpful!

**Comment by Extravi** [comment]
I think it has something like that give it a try and let me know

**Comment by amogusussy** [comment]
@DavidDev1334 This should probably be its own issue.

**Comment by Extravi** [comment]
Sorry, I'll have to look at this again at some other date.

**Pull request closed without merging**